client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/tasks.tsx

```diff
@@ -68,6 +68,18 @@ export const tasks: Task[] = [
 		taskType: 'blog',
 		disabled: true,
 	},
+	{
+		id: 'sensei_setup',
+		completed: true,
+		taskType: 'blog',
+		disabled: false,
```

Comment:
You can change this property to `disable: true` to ensure the `Set up Course Site` does not appear as if it's clickable on the launchpad screen.

Reply:
Fixed in 3bd4789eafd176d21af3091af84787085a67df85